web/packages/teleterm/src/mainProcess/agentRunner/agentRunner.test.ts

```diff
@@ -89,16 +89,9 @@ test('status updates are sent on a successful start', async () => {
     expect(agentRunner.getState(rootClusterUri)).toStrictEqual({
       status: 'not-started',
     } as AgentProcessState);
-    await new Promise((resolve, reject) => {
-      const timeout = setTimeout(
-        () => reject('Process start timed out.'),
-        4_000
-      );
```

Comment:
Race & timeout are not necessary because we're in a Jest test. It'll automatically time out after 5s.